src/cosmoz-tab-card.js

```diff
@@ -60,19 +64,25 @@ const CosmozTabCard = ({ heading }) =>
 		</style>
 
 		<div id="header" part="header">
+			${collapsable
+				? html`<iron-icon
+						icon="icons:info"
+						@click=${() => setCollapsed((c) => !c)}
+				  ></iron-icon>`
+				: nothing}
```

Comment:
It's better to use the `when` directive:

```suggestion
			${when(collapsable, () =>
				html`<iron-icon
						icon="icons:info"
						@click=${() => setCollapsed((c) => !c)}
				  ></iron-icon>`)}
```

Also, you should switch the icon based on the `collapsed` state. Or `rotate` it using CSS if you're feeling fancy.

Reply:
Done! I did use the `transform: rotate()` property for a collapsed and not collapsed card, I'm sending a video screen on Slack (I'll also post in the PR above). Probably the icon is still to be changed, but that's for the UX channel!